[INFO -- 2017-02-27 11:06:03]
Starting flip_value script
version: 2.0
[INFO -- 2017-02-27 11:06:03]
Initialization strings:
[INFO -- 2017-02-27 11:06:04]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-02-27 11:06:04]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-27 11:06:04]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-27 11:06:04]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-27 11:06:04]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-27 11:06:04]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-02-27 11:06:04]
initSignal:2
[INFO -- 2017-02-27 11:06:04]
endSignal:4
[INFO -- 2017-02-27 11:06:04]
seqSignal:30
[INFO -- 2017-02-27 11:06:04]
faultMode:3
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[EXCEPTION -- 2017-02-27 11:06:12]
pythonException: invalid frame
[INFO -- 2017-02-27 11:06:12]
Backtrace BEGIN:
#0  0x0000000000404da5 in L_lud_omp_123__par_loop2_2_5 () at lud_omp.c:153
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffd9c0 in ?? ()
#3  0x00007ffff4ea0100 in ?? ()
#4  0x0000000000000001 in ?? ()
#5  0x0000000000000003 in ?? ()
#6  0x0000000000000000 in ?? ()

148	                        sum[j] += temp_left[BS*i + k] * temp_top[BS*k + j];
149	                    }
150	                }
151	#pragma omp simd 
152	                for (j = 0; j < BS; j++) {
153	                    BB((i+i_global),(j+j_global)) -= sum[j];
154	                    sum[j] = 0.f;
155	                }
156	            }
157	        }

Backtrace END
Fliping array at pos: 3
Memory content before bitflip:0x000x000x000x00
Fault Mode: Zero bit-flip
base address to flip value: 0x7fffffffc84c
address max offset: 4
Memory content after  bitflip:0x000x000x000x00
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: sum
symbol filename: lud_omp.c
symbol line: 130
value: {0 <repeats 12 times>, 0.03413634, 0.0341022797, 0.0340680741, 0.0340340436}
value address: 0x7fffffffc840
Type: The type is an array.
Type sizeof: 64
Field name: None
Field Type: A range type, that is, an integer type with bounds.
Field Type sizeof: 1
Field Type range: (0L, 15L)

[INFO -- 2017-02-27 11:06:12]
Fault Injection Successful after 8.93103909492s
[INFO -- 2017-02-27 11:06:24]
event type: exit
[INFO -- 2017-02-27 11:06:24]
exit code: 0
